lib/rucio/daemons/conveyor/common.py

```diff
@@ -450,7 +446,7 @@ def bulk_group_transfer(transfers, policy='rule', group_bulk=200, source_strateg
     return grouped_jobs
 
 
-def get_conveyor_rses(rses=None, include_rses=None, exclude_rses=None, vos=None):
+def get_conveyor_rses(rses=None, include_rses=None, exclude_rses=None, vos=None, logger=logging.getLogger):
```

Comment:
this wouldn't work. This either needs some special handling or `logger=logging.log`.

idea for special handling (could also be done on a higher level, if useful):
```py
LOGGER = logging.getLogger(…)

def get_conveyor_rses(…, logger=LOGGER.log): 
```

Reply:
Yeah, that's just a plain typo.